src/k8s-engine/lima.ts

```diff
@@ -1246,9 +1250,9 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
    * @precondition The VM configuration is correct.
    */
   protected async startVM() {
-    this.lastCommandComment = 'Installing networking requirements';
```

Comment:
This needs a comment, along the lines of "keep displaying the status for the current action until the next command is run". Because there's an `if`-block before `startVM` is invoked.

Reply:
Actually I'm not sure exactly what I was requesting here (or the force-push commit addressed it). Looks fine now.